src/test/java/com/ncc/aif/TestUtils.java

```diff
@@ -25,11 +32,14 @@
  * Call {@link #startNewTest()} before each test to ensure a clean model.
  */
 class TestUtils {
+    protected Logger logger;
 
     // Constructor parameters
     private final ValidateAIF validator;
     private final String annotationNamespace;
-    private final boolean forceDump;
+    private final boolean dumpAlways;
+    private final boolean dumpToFile;
+    private final String dumpDirectory = "test-dump-output";
```

Comment:
Maybe add a line between dumpToFile and dumpDirectory, as the latter isn't a constructor parameter.  For that matter, it could be a local var since it's only used in one place, but I understand why you're calling it out here.  You could also make it static to make it a full-blown class constant (and probably change it to DUMP_DIRECTORY in that case).

Reply:
Fixed. Changed it to class constant DUMP_DIRECTORY.